LEAF_Request_Portal/admin/templates/mod_workflow.tpl

```diff
@@ -1877,7 +1823,18 @@ function loadWorkflow(workflowID) {
             	if(posY < minY) {
             		posY = minY;
             	}
-            	$('#workflow').append('<div class="workflowStep" id="step_'+ res[i].stepID +'">'+ res[i].stepTitle +'</div><div class="workflowStepInfo" id="stepInfo_'+ res[i].stepID +'"></div>');
+
+                let emailNotificationIcon = '';
+                if(typeof res[i].stepData == 'string') {
+                    let stepParse = JSON.parse(res[i].stepData);
+                    if(stepParse.AutomateEmailGroup === 'true'){
+                        let dayCount = stepParse.DateSelected * stepParse.DaysSelected;
```

Comment:
PSR12 requires a space between if ( and a space between ) {

Reply:
And the Linters scream bloody murder if there aren't spaces.  You would think you kicked there dog.   But in this case I agree:  Helps readability.

Reply:
I was half tempted to just use the auto formatter on this and let it eat. But I also do not want to blow up the change log.